packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -338,6 +338,21 @@ describe('Facet Test Suite', () => {
       });
     });
 
+    it('should work as expected with accessibility support', () => {
+      setupWithValues();
+      const selectedValue = 'People';
+      scope('should be selected when space is pressed', () => {
+        Actions.findSearchInputPressTabPressSpace();
+        Expect.selectedValuesContain(selectedValue);
+        Expect.numberOfSelectedLinkValues(1);
+      });
+      scope('should be selected when Enter is pressed', () => {
+        Actions.findSearchInputPressTabPressEnter();
```

Comment:
Its quit tricky to find names for these kinds of actions since testing accessibility feature is normally involved with setting focus to one element and then move focus to other element using key Tab. Under this setup, the element first focused is not the element that ends up being selected.

Any suggestions for the naming convention for these kind of Actions where key Tab is involved as we certainly need to implement similar tests moving forward.

Reply:
> Any suggestions for the naming convention for these kind of Actions where key Tab is involved as we certainly need to implement similar tests moving forward.

Personally, I find it easier to name things by thinking about the outcome. What happens when this function is called (not individual operations, but what it means for the user)?

For example, suppose we have a filter component where each value has a checkbox. The user also has to click an Apply button to make the changes effective.

In order to change the selected value our function would need to:
* uncheck all values
* check the requested value
* click the Apply button

Naming this function `deselectAllValuesThenCheckXThenApply` describes the operations that are performed by the function, but not the outcome.

In this case, the goal of this function is to replace a filter value, so naming the function `replaceFilterValue` is more meaningful. In addition it is loosely coupled to the implementation so the function name remain relevant even if the markup changes.

Regarding accessibility, we could group the accessibility actions together in an object (e.g., `A11yActions`) so the keyboard navigation would become implicit.

What do you think? Does it make sense to you?